apps/src/aichat/redux/slice.ts

```diff
@@ -183,14 +183,18 @@ const aichatSlice = createSlice({
       state.chatEventsPast = [];
       state.chatEventsCurrent = [];
     },
-    setChatMessagePending: (
-      state,
-      action: PayloadAction<PendingChatMessage>
-    ) => {
-      state.chatMessagePending = action.payload;
-      state.hasSentMessage = true;
+    updateChatMessage: (state, action: PayloadAction<ChatMessage>) => {
```

Comment:
Am I right that the thing that gets updated on a user message once we get a response is just the status (eg, profanity, etc)? For future readers, could we be more explicit here with what we want to update (vs. allowing any set of properties to be updated at once)?